### ISSUE TITLE
Search by questions returns no result for empty string even before typing the question

### ISSUE THREAD

**Issue by kunmi02** [publiclab/plots2#10882]
### Please describe the problem (or idea)

When you go to the search page and you try to select the by type dropdown then select questions, the page loads and return a no result for an empty string response to the page.

<img width="1440" alt="Screenshot 2022-03-31 at 09 42 03" src="https://user-images.githubusercontent.com/27483062/161014514-dd7d3dd8-05d6-4532-8037-8c40eb9bdb5e.png">

> What happened just before the problem occurred? Or what problem could this idea solve?

This idea will solve the issue of searching for an empty string

> What did you expect to see that you didn't?

I expect the page to only return a response when the search icon is clicked or the field is not empty

### Please show us where to look
https://publiclab.org/search/questions/

### What's your PublicLab.org username?

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by kunmi02** [comment]
Hi @TildaDares and @jywarren I found this bug on the site and I will like to work on it.

**Comment by Ifetakuro** [comment]
@kunmi02 do you mind both of us working on the bug together?

**Comment by TildaDares** [comment]
Hi @kunmi02, great catch! You can go ahead!

**Comment by girija0707** [comment]
hey @kunmi02 did you clone this repository even I would like to work on few issues

**Comment by kunmi02** [comment]
Hi @Ifetakuro,

I already wrote the fix for the issue and will be pushing out my PR soon. but if you don't mind, we can go ahead to share knowledge on how I resolved the issue.

@girija0707 Yes I did clone the repo

**Comment by girija0707** [comment]
> Hi @Ifetakuro,
> 
> I already wrote the fix for the issue and will be pushing out my PR soon. but if you don't mind, we can go ahead to share knowledge on how I resolved the issue.
> 
> @girija0707 Yes I did clone the repo

Yes I would love to know how did you resolve that issue, I'm thinking of cloning this repo but I don't have knowledge regarding Ruby hence skeptical😅

**Comment by Ifetakuro** [comment]
@kunmi02 that would be great, would love to learn

**Comment by kunmi02** [comment]
Here is the link to the PR for the https://github.com/publiclab/plots2/pull/10896

I just did a catch on the page to see if the params[:query] is not blank, if it is, I return a message asking for a search term
